include/exadg/incompressible_navier_stokes/user_interface/enum_types.cpp

```diff
@@ -59,6 +59,19 @@ enum_to_string(ProblemType const enum_type)
   return string_type;
 }
 
+void
+string_to_enum(ProblemType & enum_type, std::string const string_type)
```

Comment:
I see these changes independent of the present PR and hence they should not be included here.